CSS/contact.css

```diff
@@ -0,0 +1,168 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Roboto', sans-serif;
+    overflow-x: hidden;
+    
+}
+
+.header{
```

Comment:
I don't think it is a good idea to have a CSS file per page at this level. As you can see you are duplicating this class in about three files. And part of the expectations of the delivery is to avoid CSS style duplication.

I would advice you to merge all these files into one.